CI/linux/03_package_obs.sh

```diff
@@ -43,14 +43,15 @@ package-obs-standalone() {
     GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
     GIT_HASH=$(git rev-parse --short=9 HEAD)
     GIT_TAG=$(git describe --tags --abbrev=0)
+    UBUNTU_VERSION=$(lsb_release -sr)
 
     if [ "${BUILD_FOR_DISTRIBUTION}" ]; then
         VERSION_STRING="${GIT_TAG}"
     else
         VERSION_STRING="${GIT_TAG}-${GIT_HASH}"
     fi
 
-    FILE_NAME="obs-studio-${VERSION_STRING}-Linux.deb"
+    FILE_NAME="obs-studio-ubuntu-${UBUNTU_VERSION}-${VERSION_STRING}.deb"
```

Comment:
Prefer that the OBS Studio version string is immediately after "obs-studio" to associate the two pieces of information.

```
obs-studio-${VERSION_STRING}-ubuntu-${UBUNTU_VERSION}.deb
```

Reply:
fixed